pkg/management/authorizationdiff.go

```diff
@@ -0,0 +1,136 @@
+package management
```

Comment:
There is a lot of generic names in this file : input, map, apply but we are dealing with authorization only. I think to have less generic name could make the code easier to understand